appium/options/android/common/adb/adb_options.py

```diff
@@ -0,0 +1,367 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from datetime import timedelta
+from typing import Optional, Union
+
+from appium.options.base_options_descriptor import OptionsDescriptor
+from appium.options.common.supports_capabilities import SupportsCapabilities
+from appium.options.transformers import transform_duration_get, transform_duration_set
+
+
+class AdbExecTimeoutOption(SupportsCapabilities):
+    ADB_EXEC_TIMEOUT = "adbExecTimeout"
+    adb_exec_timeout = OptionsDescriptor[Optional[timedelta], Union[timedelta, int]]
+    (ADB_EXEC_TIMEOUT, transform_duration_get, transform_duration_set)
```

Comment:
I assume this definition is not parsed correctly being split into two lines. Write it in a single line and then `make black` to wrap it automatically. Also applicable to other classes below